app/coffeescripts/views/conferences/ConferenceView.coffee

```diff
@@ -126,3 +128,81 @@ define [
         else
           window.open(data[0].url)
       )
+
+    deleteRecording: (e) ->
+      return if !confirm(I18n.t('recordings.confirm.delete', "Are you sure you want to delete this recording?"))
+      e.preventDefault()
+      $deleteButton = $(e.currentTarget).parent()
+      @toggleActionButton($deleteButton, {state: "processing", action: "delete"})
+      @toggleRecordingLink($deleteButton, {state: "processing"})
+      $.ajaxJSON($deleteButton.data('url') + "/recording", "DELETE", {
+          recording_id: $deleteButton.data("id"),
+        }, (data) =>
+          if data.deleted == "true"
+            @removeRecordingRow($deleteButton)
+            return
+          @ensureActionPerformed($deleteButton, 1, @removeRecordingRow)
+      )
+
+    ensureActionPerformed: ($button, attempt, callback) =>
+      $.ajaxJSON($button.data('url') + "/recording", "GET", {
+          recording_id: $button.data("id"),
+        }, (data) =>
+          if $.isEmptyObject(data)
+            callback($button)
+            return
+          if attempt < 5
+            attempt += 1
+            setTimeout((=> @ensureActionPerformed($button, attempt, callback); return;), attempt * 1000)
+            return
+          $.flashError(I18n.t('conferences.recordings.action_error', "Sorry, the action performed on this recording failed. Try again later"))
```

Comment:
We don't use keys anymore with I18n so we can remove `'conferences.recordings.action_error',` here.